models/token.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+
+const MODEL = {
+    token: Joi
+        .string()
+        .token(),
+
+    id: Joi
+        .number()
+        .integer()
+        .positive(),
+
+    userId: Joi
+        .number()
+        .integer()
+        .positive()
+        .description('User ID'),
+
+    name: Joi
+        .string()
+        .max(128)
+        .description('Token name')
+        .example('Mobile token'),
+
+    description: Joi
+        .string()
+        .max(256)
+        .description('Token description')
+        .example('Used to authenticate the mobile app'),
+
+    lastUsed: Joi
+        .date()
```

Comment:
This should probably be `.string().isoDate()` to be consistent with other timestamps.
https://github.com/screwdriver-cd/data-schema/blob/master/models/build.js#L21-L25